TestViewer/MainPage.xaml.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.UI.Composition;
+using Windows.UI.Composition.Scenes;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Hosting;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using SceneLoaderComponent;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace TestViewer
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        Compositor compositor;
```

Comment:
>  Compositor compositor; [](start = 6, length = 24)

Remove any fields that are not needed. The Compositor is only used in the constructor.

Reply:
_constructor is also being used in LoadGLTF so removing this field from here would cause an error.

---
In reply to: [280579217](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280579217) [](ancestors = 280579217)